### PR TITLE
Use run id from cds

### PR DESCRIPTION
**Pull request by kunyavskiy**

### CODE REVIEW

File: src/reactions-bot/src/main/kotlin/org/icpclive/reacbot/Storage.kt [hunk 1]
```diff
@@ -47,21 +47,21 @@ class Storage {
                 )
             val voted = Votes.selectAll()
                 .where { Votes.chatId eq chatId }
-                .map { Vote.wrapRow(it).id.value }
+                .map { Vote.wrapRow(it).runId }
```
Comment (kunyavskiy) [inline src/reactions-bot/src/main/kotlin/org/icpclive/reacbot/Storage.kt:50]: It looks like there was some bug here. vote ids and run ids was messed up. But I don't really sure what's going on.

Comment (kbats183) [reply src/reactions-bot/src/main/kotlin/org/icpclive/reacbot/Storage.kt:50]: May be
I want to rewrite this code

File: src/cds/core/src/main/kotlin/org/icpclive/cds/api/OrganizationInfo.kt [hunk 2]
```diff
@@ -4,10 +4,15 @@ import kotlinx.serialization.Serializable
...
 @JvmInline
 @Serializable
-public value class OrganizationId(public val value: String) {
+public value class OrganizationId internal constructor(public val value: String) {
     override fun toString(): String = value
 }
 
+public fun String.asOrganizationId(): OrganizationId = OrganizationId(this)
+public fun Int.asOrganizationId(): OrganizationId = toString().asOrganizationId()
+public fun Long.asOrganizationId(): OrganizationId = toString().asOrganizationId()
```
Comment (kbats183) [inline src/cds/core/src/main/kotlin/org/icpclive/cds/api/OrganizationInfo.kt:13]: Why asOrganizationId inste of toOrganizationId?

**Review by kbats183 — APPROVED** [review]

File: src/cds/core/src/main/kotlin/org/icpclive/cds/api/OrganizationInfo.kt (hunk 2 above)
Comment (kunyavskiy) [reply src/cds/core/src/main/kotlin/org/icpclive/cds/api/OrganizationInfo.kt:13]: I'd say "toSmth" is conversion, while "asSmth" is a creation of a view. Here is something in between, I don't have strong intuition on what is closer.

**Pull request merged**